api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/GenericCohortQueries.java

```diff
@@ -320,4 +322,25 @@ public CohortDefinition hasNumericObs(
 
     return cd;
   }
+
+  /**
+   * Get the configurable widget parameter to be passed on the reporting UI
+   *
+   * @return
+   */
+  public Parameter getArtProgramConfigurableParameter() {
```

Comment:
@ningosi this is not a cohort function. Should this be here?

Reply:
Moved those to EptsReportUtils calss